HIP/hip-657.md

```diff
@@ -53,45 +53,38 @@ The use of the METADATA key to update NFT metadata will not affect the functiona
 ## Specification
 
 
-The METADATA key will be a new key that allows for updating the metadata of an NFT in a new NftMetadataUpdate transaction.
+The METADATA key will be a new key that allows for updating the metadata of an NFT in a new UpdateNft transaction.
 
-The NftMetadataUpdate transaction should accept the following parameters:
-	- identifier (string): the 0.0.<token-type>.<serial-no>
-- the new NFT metadata: the new metadata for the NFT
+[HIP-646](https://hips.hedera.com/hip/hip-646) and [HIP-765](https://hips.hedera.com/hip/hip-765) will add metadata on the token level. The METADATA key will also allow the metadata to be updated on the token level.
 
+The UpdateNft transaction should accept the following parameters:
 
-The NftMetadataUpdate transaction should have a new token type field called "dynamic_metadata" of type bool, which is valid only for non-fungible tokens. By default, this field is set to false (meaning the metadata for this token type is immutable). If set to true, owners of this token type can change their NFT's metadata via the NftMetadataUpdate transaction.
+- `nft_id`: the identifier (`NftID`) of the NFT to be updated
+- `metadata`: the new metadata for the NFT
 
 
 Example transaction:
 ```js
-NftMetadataUpdate()
-     .setTokenId(tokenId)
-     .setMetaData(newMetaData)
+const transaction = await TokenUpdateNft()
```

Comment:
@netopyr 
Should this just be TokenUpdate? or TokenMetadataUpdate?

When converging with all three HIPs, one is referncing the token level metadata for a FT.

I don't see a different transaction type for the FT and if we remove NFT from this defintion it should work for both token types I think. thoughts?

Reply:
I need some help understanding this comment. `TokenUpdate` allows metadata to be modified on the token level. This applies to fungible and non-fungible tokens. `TokenUpdateNfts` modifies individual NFTs. There is no equivalent functionality needed for fungible tokens.

Reply:
My thought process here is that if the partition HIP comes through with partitions on fungible tokens, and those partitions have metadata ability, we will have to make a new call to update each of those partitions if the metadata key is set. 

Conceptually, if partitions are treated as new tokens (they'll have individual entities with parent entities assigned I believe) and that is treated the same way as a token level update, then all my concerns are gone and I agree with this path.